src/components/phone-input.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { Fragment } from 'react';
+import { FormFieldErrorList } from './form-field-error-list';
+
+interface PhoneInputProps {
+  name: string;
+  label: string;
+  value: string;
+  onValueChange: (value: string) => void;
+  errors?: string[];
+}
+
+export const PhoneInput = ({ name, label, value, onValueChange, errors }: PhoneInputProps): React.ReactElement => {
+  const setPhoneValue = (phoneDigits: string) => {
```

Comment:
```suggestion
  const handleChange = (phoneDigits: string) => {
```

A gente tem uma convenção de usar uma abordagem declarativa nas nomenclaturas desse tipo de callback. O `on` já é um prefixo padronizado pelo React para nomear as props que recebem callbacks, e o `handle` é a nossa convenção para a função que implementa esse callback. O ideal é que esse handle venha acompanhado do evento que foi triggado, que pode ser o mesmo nome que a prop, ou ainda algo mais específico, caso apenas o nome da prop não seja claro.

Exemplos para esse caso: `handleChange` ou (mais específico) `handleInputChange`